packages/ckeditor5-ui/src/bindings/draggableviewmixin.ts

```diff
@@ -0,0 +1,181 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/bindings/draggableviewmixin
+ */
+
+import type View from '../view.js';
+import { type Constructor, type Mixed, global, type EventInfo } from '@ckeditor/ckeditor5-utils';
+
+/**
+ * A mixin that brings the possibility to observe dragging of the view element.
+ */
+export default function DraggableViewMixin<Base extends Constructor<View>>( view: Base ): Mixed<Base, DraggableView> {
+	abstract class DraggableMixin extends view {
```

Comment:
Nitpick: if you added `implements DraggableView`, the compiler could check the contract. Especially that you have cast to `any` at the bottom.